applications/pelion_client/README.rst

```diff
@@ -458,7 +458,7 @@ You can find the configuration files in the :file:`applications/pelion_client/co
 For each supported build target, you can find a subdirectory that contains all configuration files for the given target.
 Configuration files are provided for different `nRF Pelion Client build types`_ for each supported build target.
 
-The build types names are replacing the *${CMAKE_BUILD_TYPE}* variable in the configuration file names (for example, :file:`pm_static_ZDebug.yml`).
+The build type names are replacing the *${CMAKE_BUILD_TYPE}* variable in the configuration file names (for example, :file:`pm_static_ZDebug.yml`).
```

Comment:
Should *${CMAKE_BUILD_TYPE}* be in backticks, as it is a variable name? Right now, it only shows as italicized text in the build preview.

Reply:
It is a variable within a file name (ZDebug in the file name at the end of the line), see the list in lines 468-470, so it must be italicized.
For similar cases, check the step 3 in this section: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#building-on-the-command-line